### PR TITLE
Add kopf crd chart

### PR DESCRIPTION
**Pull request by tylerauerbeck**
#### What is this PR About?
This adds a chart that deploys the CRDs for https://github.com/nolar/kopf . It provides the ability to deploy both or only one of kopfpeering/clusterkopfpeering depending on your need.

#### How do we test this?
`helm template .` 

cc: @redhat-cop/day-in-the-life @jkupferer

### CODE REVIEW

**Review by garethahealy — CHANGES_REQUESTED** [review]
@tylerauerbeck ; just 2 requests related to testing:

- add a test to the conftest.sh ; should just be a case of duplicating a current test pointing at this new chart: https://github.com/redhat-cop/helm-charts/blob/master/_test/conftest.sh#L12-L21

- add a .test.sh to the chart ; 3 phases, install/test/clean - https://github.com/redhat-cop/helm-charts/blob/master/charts/dev-ex-dashboard/.test.sh

**Comment by tylerauerbeck** [comment]
@garethahealy This should be good now

File: _test/conftest.sh [hunk 1]
```diff
@@ -183,4 +183,15 @@ setup_file() {
 
   print_info "${status}" "${output}" "${cmd}" "${tmp}"
   [ "$status" -eq 0 ]
-}
\ No newline at end of file
+}
+
+@test "charts/kopf" {
+  tmp=$(helm_template "charts/tekton-demo")
```
Comment (garethahealy) [inline _test/conftest.sh:189]: path is wrong, would expect the test name to be same as tmp, presume copy+paste error

File: charts/kopf/.test.sh [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+trap "exit 1" TERM
+export TOP_PID=$$
+
+export project_name="kopf-$(date +'%d%m%Y')"
+
+install() {
+  echo "install - $(pwd)"
+
+  oc new-project ${project_name}
+  helm install kopf kopf
```
Comment (garethahealy) [inline charts/kopf/.test.sh:12]: is this referencing the local directory chart? or a remote? I've always done `helm install {name} .` (dot for current dir)

Comment (tylerauerbeck) [reply charts/kopf/.test.sh]: Yepp. Just the local chart. Just didn't know where this gets run from so just assumed the use of the dir name. Went ahead and fixed this.

File: _test/conftest.sh (hunk 1 above)
Comment (tylerauerbeck) [reply _test/conftest.sh]: Yepp. Copy Pasta. Fixed.

**Review by garethahealy — APPROVED** [review]

**Pull request merged**